src/main/java/uk/gov/hmcts/reform/sscs/service/EsaDecisionNoticeOutcomeService.java

```diff
@@ -61,9 +61,14 @@ public void performPreOutcomeIntegrityAdjustments(SscsCaseData sscsCaseData) {
                         }
                     }
                 }
+                if ("refused".equalsIgnoreCase(sscsCaseData.getWriteFinalDecisionAllowedOrRefused())) {
```

Comment:
might be easier to always set dwpReassessTheAward to null then do the check for not refused to set the other fields to null